src/solvers/refinement/string_constraint_generator_valueof.cpp

```diff
@@ -437,21 +439,23 @@ exprt string_constraint_generatort::add_axioms_for_parse_int(
     not_exprt(or_exprt(starts_with_minus, starts_with_plus));
 
   /// TODO: we should throw an exception when this does not hold:
-  exprt correct=add_axioms_for_correct_number_format(str, radix);
+  std::size_t max_string_length=40;
+  exprt correct=add_axioms_for_correct_number_format(
```

Comment:
This is not required here as `add_axioms_for_correct_number_format` returns a new expression